riggerlib/client.py

```diff
@@ -115,6 +115,7 @@ def terminate(self):
         try:
             self._request({'event_name': 'shutdown'})
             self.ready = False
+            self.zmq.close()
```

Comment:
But... I assume they instantiate `RiggerClient` or a subclass of it and they also call `terminate` at some point. Correct?